upgrade/test_crossgrade.sh

```diff
@@ -127,6 +127,13 @@ if [[ -z "${FROM_HUB}" || -z "${FROM_TAG}" || -z "${FROM_PATH}" || -z "${TO_HUB}
     exit 1
 fi
 
+if [[ "${TEST_SCENARIO}" == "upgrade-downgrade" || "${TEST_SCENARIO}" == "upgrade" || "${TEST_SCENARIO}" == "downgrade" ]];then
+    echo "The test scenario is ${TEST_SCENARIO}."
```

Comment:
nit: either `The current test scenario is ${TEST_SCENARIO}` or `The test scenarios is set as ${TEST_SCENARIO}`

Reply:
Done.